### PR TITLE
FileUploader. Support for custom message body and unauthorized response.

### PR DESCRIPTION
**Pull request by masterdany88**
FileUploader. Support for custom message body and unauthorized response.

### CODE REVIEW

**Comment by masterdany88** [comment]
It's continuation of:
https://github.com/GwtMaterialDesign/gwt-material-addins/pull/84

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>gwt-material-parent</artifactId>
         <groupId>com.github.gwtmaterialdesign</groupId>
-        <version>1.5.2-SNAPSHOT</version>
+        <version>1.5.1</version>
```
Comment (kevzlou7979) [inline pom.xml]: Please chnage this to 1.5.2-SNAPSHOT, we might avoid to return back to 1.5.1

Comment (BenDol) [inline pom.xml]: Just revert all the pom changes please

File: src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java [hunk 2]
```diff
@@ -20,649 +20,727 @@
...
+																													clickable: "#" + clickable,
+																													withCredentials: withCredentials
+																													});
+
+																													zdrop.on('drop', function () {
+																													that.@gwt.material.design.addins.client.fileuploader.MaterialFileUploader::fireDropEvent()();
+																													if(preview) {
+																													$wnd.jQuery(e).removeClass("active");
+																													}
+																													});
```
Comment (BenDol) [inline src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java]: remove the new lines here

**Comment by BenDol** [comment]
There are too many issues with this PR, please resolve the following:
* Use 4 spaces for tabulation instead of actual tabs
* Please revert the pom.xml changes
* Remove the large amount of new lines in MaterialFileUploader.java

Will close this for now so it can be redone (sorry that this is taking a whole, but it needs to be done right).

Thanks for sticking at it! We appreciate your help :+1:

File: src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java (hunk 2 above)
Comment (masterdany88) [inline src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java]: The problem is that my eclipse is doing this spaces here. I do not know how to change it. I would like to use same ide and same configuration as You use. Can You share spec of Your IDE?

**Pull request closed without merging**